### PR TITLE
OCPCLOUD-2193: Update k8s to 1.28

### PR DESCRIPTION
**Pull request by racheljpg**
Updates kubernetes dependencies to 1.28.

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -202,9 +209,9 @@ func main() {
 // namespace.
 func newConfigManagedClient(mgr manager.Manager) (runtimeclient.Client, manager.Runnable, error) {
 	cacheOpts := cache.Options{
-		Scheme:     mgr.GetScheme(),
-		Mapper:     mgr.GetRESTMapper(),
-		Namespaces: []string{awsclient.KubeCloudConfigNamespace},
+		Scheme:            mgr.GetScheme(),
+		Mapper:            mgr.GetRESTMapper(),
+		DefaultNamespaces: map[string]cache.Config{},
```
Comment (odvarkadaniel) [inline cmd/manager/main.go:214]:
```suggestion
		DefaultNamespaces: map[string]cache.Config{
		    awsclient.KubeCloudConfigNamespace: {},
		},
```

I think you omitted the namespace in there.

**Review by odvarkadaniel — COMMENTED** [review]
Just one thing, otherwise looks good.

File: cmd/manager/main.go (hunk 1 above)
Comment (racheljpg) [reply cmd/manager/main.go]: You're right! Thanks, added the namespace back

**Pull request merged**